README.md

```diff
@@ -1,7 +1,64 @@
 # CommonFactory
- A description of what is it and what it is used for.  
-  
-  ## Usage 
-  How to use the CommonFactory script
-  ## APIs
-  - APIs list
+Package with helper scripts to deploy a Common DAO using Arc `DAOFactory`.
+
+# Usage
+Run: `npm install --save @daostack/commonfactory`
+
+Import the `commonfactory` methods: `getForgeOrgData, getSetSchemesData` and use them to generate call data to Arc `DAOFactory` methods.
+```javascript
+const { getForgeOrgData, getSetSchemesData } = require('@daostack/commonfactory');
+
+// ...
+const forgeOrg = await daoFactory.methods.forgeOrg(
```

Comment:
perhaps we can also explain here where the ABI lives (i.e. how to construct the daoFactory object); and where it is deployed

Reply:
This stuff belongs to the migration...